GoogleDataTransport/GDTCCTLibrary/GDTCCTUploader2.m

```diff
@@ -96,7 +96,21 @@ - (void)uploadTarget:(GDTCORTarget)target withConditions:(GDTCORUploadConditions
     if (weakOperation.uploadAttempted) {
       // Ignore all upload requests received when the upload was in progress.
       [weakSelf.uploadOperationQueue cancelAllOperations];
+
+      // TODO: Should we reconsider GDTCCTUploadCompleteNotification? Maybe a completion handler
+      // instead?
+#if !NDEBUG
```

Comment:
NDEBUG testing is a bit worrisome to me since I'm not sure our test infra has good coverage with and without it being set.